src/database/models/Youfocus/Playlist.ts

```diff
@@ -0,0 +1,26 @@
+import { Schema, model, models, Model } from 'mongoose';
+import {PlaylistModel} from '@/interfaces';
+
+// Define the schema
+const PlaylistSchema = new Schema<PlaylistModel>(
+  {
+    playlistUrl: {
+      type: String,
+      required: [true, 'Playlist URL is required'],
+    },
+    videos: [
+      {
+        title: String,
+        videoId: String,
+        thumbnail: String,
+      },
+    ],
+  },
+  { timestamps: true }
+);
```

Comment:
Add more fields like Playlist Name and other details. Look at the Design. We need to store all the details